huaweicloud/services/cc/data_source_huaweicloud_cc_connections.go

```diff
@@ -0,0 +1,324 @@
+package cc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API CC GET /v3/{domain_id}/ccaas/cloud-connections
+// @API CC POST /v3/{domain_id}/ccaas/cloud-connections/filter
+func DataSourceCcCloudConnections() *schema.Resource {
```

Comment:
other func names alse need to updated
```suggestion
func DataSourceCloudConnections() *schema.Resource {
```

Reply:
fixed